jwst/ami/oifits.py

```diff
@@ -367,8 +367,18 @@ def make_oifits(self):
         v2coord = self.tuv[:, 1, 0]
         u2coord = self.tuv[:, 1, 1]
 
+        # quads have 3 vectors
+        v1coord_q4 = self.quads[:, 0, 0]
+        u1coord_q4 = self.quads[:, 0, 1]
+        v2coord_q4 = self.quads[:, 1, 0]
+        u2coord_q4 = self.quads[:, 1, 1]
+        v3coord_q4 = self.quads[:, 2, 0]
+        u3coord_q4 = self.quads[:, 2, 1]
```

Comment:
This variable redefinition appears a bit unnecessary.  These variables are only used when assigning them to columns of the q4 table starting on what's currently line 577.  It'd be more compact and easier to read to just assign them directly on that line, without going through this additional variable